scanpipe/models.py

```diff
@@ -416,9 +416,6 @@ class Project(UUIDPKModel, ExtraDataFieldMixin, models.Model):
         ),
     )
 
-    class Meta:
-        ordering = ["-created_date"]
```

Comment:
What's the reason for changing the model?

Reply:
I thought to change it in the views, will put it back